docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   boulder:
-    image: &boulder_image letsencrypt/boulder-tools:${BOULDER_TOOLS_TAG:-go1.16.6_2021-07-12}
+    image: &boulder_image andygabby/boulder-tools:${BOULDER_TOOLS_TAG:-go1.16.6_2021-07-12}
```

Comment:
Don't think this is intended to be here :)

Reply:
yup, noticed it when tests failed. Already fixed. Thanks!